stingray/pulse/search.py

```diff
@@ -5,7 +5,10 @@
 from ..utils import contiguous_regions
 from astropy.stats import poisson_conf_interval
 import matplotlib.pyplot as plt
-
+from stingray import EventList
```

Comment:
I think this PulsarPeriodogram should only take times, not event lists, as input. This frees us from this potentially circular import here.